### DISCUSSION TITLE
How to deploy Psi framework on ARM-linux board?

### DISCUSSION THREAD

**Discussion by HaipingLe** [microsoft/psi#196] [General]
Does Microsoft have any documents to describe how to porting the Psi to ARM-Linux platform?

**Comment by AshleyF** [comment]
The core \psi runtime works anywhere that .NET Standard is supported. This includes Linux and ARMv7+. For example, I personally run it on Raspberry Pi boards (except the Zero, which is ARMv6) under Ubuntu Edge and Raspian. The pre-built NuGets work on Linux and there are also [instructions here](https://github.com/microsoft/psi/wiki/Building-the-Codebase#on-linux) describing how to build the codebase on Linux. As you [peruse the NuGets](https://www.nuget.org/packages?q=Microsoft.Psi) and projects, you’ll notice some Linux-specific (ending in `.Linux`) and some Windows-specific (ending in `.Windows`). Those with no suffix work cross-platform.

Hope this helps!

**Reply by HaipingLe** [reply]
Thank you very much.
